DevEdu.API/DevEdu.API/Models/InputModels/LessonInputModel.cs

```diff
@@ -8,9 +8,15 @@ public class LessonInputModel
     {
         [Required(ErrorMessage = DateRequired)]
         public DateTime Date { get; set; }
+
```

Comment:
in input model the Date property must be a string
and we must convert it to DateTime in MapperProfile